pkg/cmd/pulumi/pulumi.go

```diff
@@ -649,5 +649,6 @@ func confirmPrompt(prompt string, name string, opts display.Options) bool {
 
 	reader := bufio.NewReader(os.Stdin)
 	line, _ := reader.ReadString('\n')
-	return strings.TrimSpace(line) == name
+	input := strings.TrimSpace(line)
+	return strings.Trim(input, "\"") == name
```

Comment:
I worry about putting this so deep in input parsing, there _may_ be cases where '"' is a valid char to input and this will confuse the hell out of us if we forget about it.
Also are they're really enough people adding quotes to their prompt answers that this is needed?

Reply:
OK I re-read the issue, this is _very_ specific to "pulumi stack rm" so I'd be ok with the trim('"') there.

Reply:
I'll recheck usage and name this explicitly for stackRm if it's the only user.